packages/Checkbox/stories/Checkbox.stories.styles.js

```diff
@@ -0,0 +1,8 @@
+import styled from "styled-components";
+import { Story } from "storybook/assets/styles/common.styles";
+
+// Common Checkbox story styles
+
+export const CheckboxStory = styled(Story)`
```

Comment:
Just my opinion that this file adds unnecessary noise

Reply:
We can refactor this out when we create some common storybook components and approaches